apps/archive/archive.py

```diff
@@ -303,7 +303,10 @@ def _handle_media_updates(self, updates, original, user):
             self._set_association_timestamps(item_obj, updates, new=False)
 
             stored_item.update(item_obj)
-            updates[ASSOCIATIONS][item_name] = stored_item
+            if 'test' in item_name:
```

Comment:
On settings you can define multiple related items fields, I think 'test' is just the name of one of defined related items on your instance.
You have to get the list of related items fields and check them against the fields on associations.
The key on associations has field_name--number format